src/modules/components/headers-and-footers/footer/footer.component.html

```diff
@@ -12,7 +12,8 @@
         <h2 class="visuallyhidden">Support links</h2>
 
         <ul>
-          <li data-ng-repeat="link in $ctrl.links"><a href="{{ link.url }}" data-ng-bind="link.name"></a></li>
+          <li data-ng-repeat="link in $ctrl.links"><a href="{{ link.url }}" data-ng-bind="link.name" target="{{link.target}}">
```

Comment:
Instead of having to pass the target _self each time, I'd use a default option in the option, something like:
target="{{link.target || '_self' }}"